docs/docs-components/AppLayout.js

```diff
@@ -18,11 +19,18 @@ type Props = {|
 
 export default function AppLayout({ children, colorScheme, isHomePage }: Props): Node {
   const { isMobile } = useDocsDeviceType();
-  const { isSidebarOpen } = useNavigationContext();
+  const { isSidebarOpen, setIsSidebarOpen } = useNavigationContext();
 
   const footerColor =
     colorScheme === 'dark' ? 'var(--color-gray-roboflow-700)' : 'var(--color-orange-firetini-0)';
 
+  useEffect(() => {
+    if (typeof window !== 'undefined')
+      window.addEventListener('scroll', () => setIsSidebarOpen(false));
+
+    return () => window.removeEventListener('scroll', () => setIsSidebarOpen(false));
```

Comment:
This isn't actually removing the event listener — these are independent anonymous functions, with different references. You'll need to name the function to be able to remove it:
```js
useEffect(() => {
  const handleScroll = () => setIsSidebarOpen(false);
  window.addEventListener('scroll', handleScroll);
  return () => {
    window.removeEventListener('scroll', handleScroll);
  };
});
```

Reply:
Thanks! good catch

Reply:
DONE